src/components/MessageSender.js

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export default function MessageSender() {
-	return <Wrapper>메시지 전송부</Wrapper>;
+export default function MessageSender({message, changeNewMsg, updateMsglist}) {
+	
+	return (
+		<Wrapper>
+			<form onSubmit={updateMsglist}>
+				<input onChange={changeNewMsg} value={message}></input>
+				<button type="submit">전송</button>
+			</form>
+		</Wrapper>
+	);
 }
-const Wrapper = styled.div``;
+const Wrapper = styled.div`
+	box-sizing: border-box;
+	width: 100%;
+	position: fixed;
+	bottom: 0;
+	background-color: rgba(255,255,255,0.8);
+	padding: 10px;
+	form {
+		display: flex;
+		flex-direction: row;
+	}
+	input {
+		box-sizing: border-box;
+		background-color: rgba(0,0,0,0);
+		height: 50px;
+		border: none;
+		margin: 0 10px 0 0;
+		padding: 10px;
+		flex: 1;
+		font-size: 15px;
+		border: 1px solid rgba(0, 0, 0, 0.3);
+		border-radius: 20px;
+	}
+	input: focus{
+		outline: none;
+	}
```

Comment:
SCSS, styled-components에서 `&`을 이용하면 현재 지정된 엘레멘트들을 지칭할 수 있답니다~
(물론 일반 css는 안돼요 ㅠㅠ)

```suggestion
	input {
		box-sizing: border-box;
		background-color: rgba(0,0,0,0);
		height: 50px;
		border: none;
		margin: 0 10px 0 0;
		padding: 10px;
		flex: 1;
		font-size: 15px;
		border: 1px solid rgba(0, 0, 0, 0.3);
		border-radius: 20px;
		
		&:focus {
		        outline: none;
		}
	}
```

Reply:
&를 이런 식으로 쓰는 거였군요!!! 감사합니다 ㅎㅎ